crates/iota-faucet/src/faucet/simple_faucet.rs

```diff
@@ -131,16 +136,19 @@ impl SimpleFaucet {
             config.max_request_queue_length as usize,
         );
 
-        // This is to handle the case where there is only 1 coin, we want it to go to
-        // the normal queue
-        let split_point = if coins.len() > 10 {
-            coins.len() / 2
+        // Split the coins eventually into two pools: one for the gas pool and one for
+        // the batch pool. The batch pool will only be populated if the batch feature is
+        // enabled.
+        let split_point = if config.batch_enabled {
+            if coins.len() > 1 {
+                1 // At least one coin goes to the gas pool the rest to the batch pool
```

Comment:
but this makes it such that no more than 1 coin goes to the gas pool if the batch pool is enabled, right? Do we want that?

Reply:
That is correct. I assumed that if we activate batch, we really want to use it and therefore provide more coins for it. Also, we should keep in mind that in the long run, the batch implementation is supposed to replace the old queue.
In case we don't want to use batch, just deactivate the batch flag and all coins are allocated to the old queue.